pkg/server/server.go

```diff
@@ -111,6 +112,37 @@ func (s *PlatformHealthServer) Check(ctx context.Context, req *ph.HealthCheckReq
 
 	providerServices := s.Config.GetInstances()
 
+	componentPath := strings.Split(req.Component, "/")
```

Comment:
`/` as separate isn't ideal, as the kubernetes provider is using it within the constructed name of its instances.